data_steward/tools/create_tier.py

```diff
@@ -0,0 +1,145 @@
+# Python imports
+import argparse
+import logging
+import re
+
+# Third party imports
+
+# Project imports
+from utils import pipeline_logging
+
+LOGGER = logging.getLogger(__name__)
+
+TIER_LIST = ['controlled', 'registered']
+DEID_STAGE_LIST = ['deid', 'base', 'clean']
+
+
+def validate_tier_param(tier):
+    """
+    helper function to validate the tier parameter passed is either 'controlled' or 'registered'
+
+    :param tier: tier parameter passed through from either a list or command line argument
+    :return: nothing, breaks if not valid
+    """
+    if tier.lower() not in TIER_LIST:
+        LOGGER.error(
+            f"Parameter ERROR: {tier} is an incorrect input for the tier parameter, accepted: controlled or "
+            f"registered")
```

Comment:
ok.  another thing, consider moving your message into a string variable, then you can send the message to the logger and raise the error.  For example,

``` 
if tier.lower() not in TIER_LIST:
    msg = "Bad argument". # your message is better, this is for example.
    LOGGER.error(msg)
    raise argparse.ArgumentTypeError(msg)
```

The empty `argparse.ArgumentTypeError` is functionally correct.  Adding the message will be more informative to whoever is running the script.

It's a difference of:
```
>>> import argparse
>>> raise argparse.ArgumentTypeError
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
argparse.ArgumentTypeError
>>> raise argparse.ArgumentTypeError('foo')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
argparse.ArgumentTypeError: foo
```
The second one would provide a more informative console message.

Reply:
Awesome, thank you for this suggestion, much better console message.